addons/account_peppol/wizard/peppol_registration.py

```diff
@@ -0,0 +1,272 @@
+
+    @api.depends('peppol_eas', 'peppol_endpoint')
+    def _compute_peppol_warnings(self):
+        for wizard in self:
+            peppol_warnings = {}
+            if (
+                wizard.peppol_eas
+                and wizard.peppol_endpoint
+                and not wizard.company_id._check_peppol_endpoint_number(warning=True)
+            ):
+                peppol_warnings['company_peppol_endpoint_warning'] = {
+                    'message': _("The endpoint number might not be correct. "
+                                "Please check if you entered the right identification number."),
+                }
+            if wizard.company_id.country_code == 'BE' and wizard.peppol_eas not in (False, '0208'):
+                peppol_warnings['company_peppol_eas_warning'] = {
+                    'message': _("The recommended EAS code for Belgium is 0208. "
+                                "The Endpoint should be the Company Registry number."),
+                }
+            wizard.peppol_warnings = peppol_warnings or False
+
+    @api.depends('account_peppol_edi_user_id')
+    def _compute_account_peppol_mode_constraint(self):
+        mode_constraint = self.env['ir.config_parameter'].sudo().get_param('account_peppol.mode_constraint')
+        trial_param = self.env['ir.config_parameter'].sudo().get_param('saas_trial.confirm_token')
+        self.account_peppol_mode_constraint = trial_param and 'demo' or mode_constraint or 'prod'
+
+    @api.depends('account_peppol_edi_user_id')
+    def _compute_account_peppol_edi_mode(self):
+        edi_mode = self.env['ir.config_parameter'].sudo().get_param('account_peppol.edi.mode')
+        for wizard in self:
+            if wizard.account_peppol_edi_user_id:
+                wizard.account_peppol_edi_mode = wizard.account_peppol_edi_user_id.edi_mode
+            else:
+                wizard.account_peppol_edi_mode = edi_mode or 'prod'
+
+    def _inverse_account_peppol_edi_mode(self):
+        for wizard in self:
+            if not wizard.account_peppol_edi_user_id and wizard.account_peppol_edi_mode:
+                self.env['ir.config_parameter'].sudo().set_param('account_peppol.edi.mode', wizard.account_peppol_edi_mode)
+                return
+
+    # -------------------------------------------------------------------------
+    # BUSINESS ACTIONS
+    # -------------------------------------------------------------------------
+
+    def _action_open_peppol_form(self, reopen=True):
+        action_dict = {
+            'name': _("Send via Peppol"),
+            'type': 'ir.actions.act_window',
+            'view_mode': 'form',
+            'res_model': 'peppol.registration',
+            'target': 'new',
+        }
+
+        if reopen:
+            action_dict.update({
+                'res_id': self.id,
+                'context': {'disable_sms_verification': True},
```

Comment:
keep active_ids/active_model:
```suggestion
                'context': {**self.env.context, 'disable_sms_verification': True},
```

Then you can reopen the send & print later